### DISCUSSION TITLE
Auto Complete in Advanced PDFs

### DISCUSSION THREAD

**Discussion by TechZack** [burkybang/SuiteAdvanced-Feedback#173] [Ideas]
Hey there, 

I have seen this feature in NetSuite Utils made by George Harrington, but it allowed you to essentially auto complete fields in Advanced PDFs. (Though I have just attempted to use it today and it seems to be broken).

Unfortunately it is impossible to run both NetSuite Utils and SuiteAdvanced at the same time as they collide with each other making client side changes.

This would be a nice feature if it is possible, making Advanced PDFs can be very tedious at times with NetSuite so anything like this to make Quality of Life better would be amazing!

**Comment by johnfinlay** [comment]
Code highlighting would be nice too.
